stylus/components/button.styl

```diff
@@ -46,7 +67,8 @@ $button
             top        em(-1px)
 
 
-$button--regular
+$button--regular // depreciated
```

Comment:
s/depreciated/deprecated/

Reply:
Oh man, it sounded like depreciated in my head so I wrote it that way, thanks.